### DISCUSSION TITLE
Meeting location now TBD with call note

### DISCUSSION THREAD

**Discussion by DCO84770** [code4recovery/12-step-meeting-list#927] [Q&A] [answered]
A meeting we have listed is still going to occur but the location is now unknown. We want people to call for the current location. See:
https://dixieaaoffice.com/meetings/?weekday=0&meeting=dixie-group

Removing the old street address prevents the meeting being listed.

Is there a better way to handle this?
Thanks!

**Comment by joshreisner** [comment, marked as answer]
We recommend taking the listing down until there's a stable meeting place, or setting it as "unable to attend in person". Providing an incorrect address, even with a note, is liable to confuse some users.

**Reply by DCO84770** [reply]
Thanks!
